src/utils/domain.ts

```diff
@@ -61,9 +65,17 @@ export const getAvatarPath = (avatar?: string, placeholder?: string) => {
 };
 
 export const getAvatarPathWithFallback = (avatar?: string, name?: string) => {
-  const placeholder = name
-    ? `https://ui-avatars.com/api/?name=${encodeURIComponent(name ?? '')}`
-    : AVATAR_PLACEHOLDER;
+  const placeholder = name ? getInitialsUrl(name) : AVATAR_PLACEHOLDER;
 
   return avatar ? getAvatarPath(avatar) : placeholder;
 };
+
+export const getCircleAvatar = ({
+  avatar,
```

Comment:
Which one is better: `avatar` or `logo`?

Reply:
I could go either way, in the future we can have Hasura do things like this. I like avatar because it is consistent and the user and circle are handled the same way generally.

Reply:
Maybe we could refactor and use a different name (although I can't think of one that fits both cases well)